src/App.tsx

```diff
@@ -1,14 +1,21 @@
 /* eslint-disable max-len */
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 
 import { TodoList } from './components/TodoList';
-import { TodoFilter } from './components/TodoFilter';
-import { TodoModal } from './components/TodoModal';
```

Comment:
You've changed the logic of the project.
Previously - TodoList , TodoFilter, TodoModal were imported into APP,
Now - TodoModal is imported into TodoList,
             TodoList is imported into the EPP,
             TodoFilter is not used - layout block added to TodoList.
As I understand, we shouldn't change logic. Maybe I wrong